kv/txn_coord_sender.go

```diff
@@ -396,14 +396,26 @@ func (tc *TxnCoordSender) sendOne(ctx context.Context, call proto.Call) {
 				return
 			}
 			tc.Lock()
-			if txnMeta := tc.txns[id]; id != "" && txnMeta != nil {
+			txnMeta, metaOK := tc.txns[id]
```

Comment:
slight preference for
```go
txnMeta := tc.txns[id]
metaOK := txnMeta != nil // don't even really need this
```

since the current version has a (theoretically) unhandled case in which there is a `nil` entry in the map.